Dockerfiles/cpp/.devcontainer/Dockerfile

```diff
@@ -19,17 +19,10 @@
 FROM --platform=$BUILDPLATFORM ubuntu:22.04 as builder
 
 RUN apt-get update && apt-get upgrade -y && \
-        apt-get install -y apt-utils openssh-client gnupg2 dirmngr iproute2 \
-        procps lsof htop net-tools psmisc manpages manpages-dev \
-        curl wget ca-certificates unzip init-system-helpers \
-        bzip2 zip git zsh pigz jq \
-        apt-transport-https dialog libkrb5-3 \
-        libgssapi-krb5-2 libicu[0-9][0-9] liblttng-ust[0-9] \
-        locales sudo ncdu cmake \
-        build-essential build-essential \
-        gcc-aarch64-linux-gnu g++-aarch64-linux-gnu binutils-aarch64-linux-gnu \
-        gcc-x86-64-linux-gnu g++-x86-64-linux-gnu binutils-x86-64-linux-gnu && \
-        apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
+    apt-get install -y jq sudo curl wget git cmake build-essential \
```

Comment:
do we need to manually install `git` if it is shipped via devcontainer feature?

Reply:
unfortunately yes, due to the python requirements. I tried to shift this installation to be done after the feature stuff but this was not working out really...

Reply:
But the feature is needed, to install the latest version of git, ubuntu just has version 2.34...

Reply:
OK